api/src/controllers/mine-controller.js

```diff
@@ -72,7 +72,7 @@ exports.protectedPut = async function (args, res, next) {
 
   const sanitizedObj = PutUtils.validateObjectAgainstModel(MineBCMI, incomingObj);
 
-  if (!sanitizedObj || sanitizedObj === {}) {
+  if (!sanitizedObj || Object.keys(sanitizedObj).length === 0) {
```

Comment:
unrelated minor fix. as js compares objects by reference, "sanitizedObj === {}" is always false.